armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -365,7 +365,12 @@ def test_findWithMinMax(self):
         self.assertIsNone(assem)
 
     def runShuffling(self, fh):
-        """Shuffle fuel and write out a SHUFFLES.txt file."""
+        """Shuffle fuel and write out a SHUFFLES.txt file.
+
+        .. test:: There is a user friendly shuffle logic interface.
+            :id: T_ARMI_SHUFFLE0
+            :tests: R_ARMI_SHUFFLE
+        """
```

Comment:
So this is kinda funny/odd since this isn't a test per se, but rather is part of the `test_repeatShuffles` test. That said, I think it would be better to have the test tag just on `test_repeatShuffles` and remove this here. Thoughts?